wordpress-mobile/WordPress-Android
# Updated constraintlayout version to fix #21429

Fixes #21429

This PR fixes a crash that was introduced by a recent `ConstraintLayout` update. The fix was simply to update that library again. Note that I had to fix a "private resource" lint error caused by `DomainItem` for the app to build.

To test:

* Using `trunk` go to "My Site," tap the ☰ icon on the Blaze card, and select "Learn more"
* Notice the app crashes
* Using this branch, do the same and note the crash is resolved
